proto/osmosis/incentives/gauge.proto

```diff
@@ -41,4 +41,57 @@ message LockableDurationsInfo {
     (gogoproto.stdduration) = true,
     (gogoproto.moretags) = "yaml:\"lockable_durations\""
   ];
+}
+
+// HistoricalReward stores the accumulator value for rewards
```

Comment:
Can we add a comment for how we're indexing this in state? Presumably its by lockup duration + lockup asset? (Don't care about the details of the key format, just the fields we index it on)